media-gfx/noteshrink/noteshrink-0.1.1.recipe

```diff
@@ -3,28 +3,55 @@ DESCRIPTION="Noteshrink is an utility that is used to convert scans of notes to
 HOMEPAGE="https://github.com/mzucker/noteshrink"
 COPYRIGHT="2016 Matt Zucker"
 LICENSE="MIT"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="https://github.com/mzucker/noteshrink/archive/$portVersion.tar.gz"
 CHECKSUM_SHA256="2a29c09768973e688b693b61337c6c49384e8123cf88824dba335cc8c4ed2ca8"
 
-ARCHITECTURES="!any"
-# Doesn't work on gcc2hybrid - missing dependencies.
+ARCHITECTURES="any"
+# The package is "any" because the result is only a Python script under /bin/.
+# However, on gcc2 systems, it will depend on Python modules only available for _x86.
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	hostArchSuffix="_x86"
+else
+	hostArchSuffix="$secondaryArchSuffix"
```

Comment:
when not using SECONDARY_ARCHITECTURES, $secondaryArchSuffix is always empty, so this could also be

```suggestion
	hostArchSuffix=""
```

Reply:
On the other hand, this is not good anyway for potentially reusing any-arch packages (see https://github.com/haikuports/haikuporter/issues/234).

Better would be if numpy, pillow and scipy added a suffix-less PROVIDE for x86_gcc2.

Reply:
It is a good thing that this is an "any" package, I'm not disputing that. Actually, anything that can be "any" should be.

But when using "any", it should be possible that, for example, you build the package on x86_64, take the resulting hpkg file and install it on x86_gcc2 (or riscv64 even), and it should work, or the other way round.

Reply:
Indeed hostArchSuffix should be dropped, if REQUIRES can be made arch agnostic.